examples/axual/22_streams.tf

```diff
@@ -1,17 +1,28 @@
-resource "axual_stream" "gitops_test_stream2" {
-  name = "gitops_test_stream2"
+resource "axual_stream" "logs" {
```

Comment:
Can we add some comments on what this topic is about? Or just set the description?